packages/template-retail-react-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@salesforce/retail-react-app",
-  "version": "1.0.0",
+  "version": "1.0.1-preview.0",
```

Comment:
For convenience, this single PR would release preview versions for _all_ public packages (i.e. SDK, commerce-sdk-react, and retail-react-app).